src/css/about_us.css

```diff
@@ -0,0 +1,119 @@
+.about {
+  padding-top: 80px;
+}
+
+.about .section-title::after {
+  display: none;
+}
+
+.about .section-title span {
+  position: relative;
+}
+
+.about .section-title span::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: calc(100% - 14px);
+  width: 100%;
+  height: 12px;
+  border-radius: 8px;
+  background-color: var(--lighter-green);
+  z-index: -1;
+}
```

Comment:
Could you please move this to base.css and replace existing solution with yours, thanks ^^)

Reply:
done